content/Project-Goals-2025-March-Update.md

```diff
@@ -162,22 +166,22 @@ Updates from [today's meeting](https://hackmd.io/@rust-lang-team/H1hZmpW6ke):
 * While not used in RFL, `naked_asm` is not on the list but it will be moving forward for stabilization. It suffers from the same LLVM bug as `global_asm` forgetting target feature flags.
 
 ### ABI-modifying compiler flags
-* Andrew Zhogin has opened a draft PR (https://github.com/rust-lang/rust/pull/138736) following Alice's issue about which santisers should be modifiers (https://github.com/rust-lang/rust/issues/138453)
+* Andrew Zhogin has opened a draft PR (<https://github.com/rust-lang/rust/pull/138736> following Alice's issue about which santisers should be modifiers (<https://github.com/rust-lang/rust/issues/138453>)
```

Comment:
accidentally deleted the closing parenthesis around the first URL here?

Reply:
Good spot. Fixed.